test/functional/gridfs_stream.test.js

```diff
@@ -95,6 +95,38 @@ describe('GridFS Stream', function () {
     }
   });
 
+  it('destroy publishes provided error', {
```

Comment:
What's this test here doing?

Reply:
@nbbeeken I think this test was here already, but it got scooped into this PR after we rebased onto the most recent 4.0 branch. We may have some git-fu to figure out to get rid of that change in this PR.

Reply:
Can we leave this comment open until the mysterious test is taken care of?